content/ember-data-polaris-2023-04-11.md

```diff
@@ -0,0 +1,76 @@
+---
+title: EmberData Polaris Update
+authors:
+  - chris-thoburn
+  - the-ember-data-team
+date: 2023-04-12T00:00:00.000Z
```

Comment:
```suggestion
date: 2023-04-14T00:00:00.000Z
```

Reply:
> You might want to remove the date from the file name.

the other file names all use a date though, and better for consistency. Happy to update the date if we care strongly that it matches the publish vs the create data.